storybook/storybook-react/src/Dialog/Dialog.stories.tsx

```diff
@@ -0,0 +1,128 @@
+        met het formulier.
+      </Paragraph>,
+    ],
+    actions: [
+      <>
+        <Button key={1} variant="tertiary" autoFocus onClick={() => document.querySelector('dialog')?.close()}>
+          Terug
+        </Button>
+        <Button key={2} type="submit">
+          Verder
+        </Button>
+      </>,
+    ],
+  },
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
+} satisfies Meta<typeof Dialog>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    open: true,
+  },
+  render: (args) => (
+    <div style={{ minHeight: '500px' }}>
+      <Dialog {...args} />
+    </div>
+  ),
+}
+
+export const ScrollContent: Story = {
+  args: {
+    open: true,
+    title: 'Privacyverklaring gemeente Amsterdam',
+    children: [
+      <>
+        <Heading size="level-4">Algemeen</Heading>
+        <Paragraph>
+          De gemeente Amsterdam verwerkt bij de uitvoering van haar taken en verplichtingen persoonsgegevens. De manier
+          waarop de gemeente Amsterdam om gaat met persoonsgegevens is vastgelegd in het stedelijk kader verwerken
+          persoonsgegevens.
+        </Paragraph>
+        <Paragraph>
+          Deze verklaring geeft aanvullende informatie over de omgang met persoonsgegevens door de gemeente Amsterdam en
+          over uw mogelijkheden tot het uitoefenen van uw rechten met betrekking tot persoonsgegevens.
+        </Paragraph>
+        <Paragraph>
+          Meer specifieke informatie over privacy en de verwerking van persoonsgegevens door de gemeente Amsterdam kunt
+          u op de hoofdpagina vinden.
+        </Paragraph>
+        <Paragraph>
+          Vanwege nieuwe wetgeving of andere ontwikkelingen, past de gemeente regelmatig haar processen aan. Dit kunnen
+          ook wijzigingen zijn in de wijze van het verwerken van persoonsgegevens. Wij raden u daarom aan om regelmatig
+          deze pagina te bekijken. Deze pagina wordt doorlopend geactualiseerd.
+        </Paragraph>
+        <Heading size="level-4">Geldende wet- en regelgeving en reikwijdte</Heading>
+        <Paragraph>
+          Vanaf 25 mei 2018 is de Algemene verordening gegevensbescherming (Avg) van toepassing op alle verwerkingen van
+          persoonsgegevens. Deze Europese wetgeving heeft directe werking in Nederland. Voor die zaken die nationaal
+          geregeld moeten worden, is de Uitvoeringswet Avg in Nederland aanvullend van toepassing. Deze wetteksten kunt
+          u vinden op de website van Autoriteit Persoonsgegevens.
+        </Paragraph>
+      </>,
+    ],
+  },
+  render: (args) => (
+    <div style={{ minHeight: '100vh' }}>
+      <Dialog {...args} />
+    </div>
+  ),
+}
+
+export const ShowDialog: Story = {
+  args: {
+    id: 'showdialog',
+    actions: [
+      <>
+        <Button
+          key={1}
+          variant="tertiary"
+          autoFocus
+          onClick={() => document.querySelector('#showdialog' as any)?.close()}
```

Comment:
This results in `Argument of type 'HTMLDialogElement' is not assignable to parameter of type 'string'.` I solved it in a different way for the close button, but the button outside the Dialog targeting an ID (because there a multiple Dialog elements in de SB Docs) needs the `any`. Typescript is "really" great ;)

Reply:
Are you sure? You might've missed the parenthesis. This works for me

Reply:
Ah right, I changed it but for the showModal click.